### DISCUSSION TITLE
关于新增截图到png

### DISCUSSION THREAD

**Discussion by Ashuknowwut** [lisai9093/YYS#25] [Q&A] [answered]
我本来想在这个脚本基础上添加一些自定义功能（比如个人突破卡57），但是发现新增的截图怎样都无法识别到。 求解答TT

**Comment by lisai9093** [comment]
你用的截图是从模拟器直接导出的吗？因为模拟器会缩放画面，所以最好是从安卓/模拟器直接导出然后裁剪才可以的

**Reply by Ashuknowwut** [reply, marked as answer]
之前是用的15debug然后再微信截的图， 改天试试从模拟器导出，谢谢！

**Reply by Ashuknowwut** [reply]
problem solved
